### PR TITLE
docs: add riot ecosystem plugin

### PR DESCRIPTION
**Pull request by ariesjia**

### CODE REVIEW

File: docs/ecosystem-riot-testing-library.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+afterEach(() => {
+  cleanup()
+})
+
+test('should show count text  when rendered', () => {
+  const { queryByTestId } = render(TestTag, {count: 10});
+  expect(queryByTestId('count').textContent).toBe("10");
+})
+
+test('should add count when click add button text', async () => {
```
Comment (kentcdodds) [inline docs/ecosystem-riot-testing-library.md]: Does this test need to be `async`?

File: docs/ecosystem-riot-testing-library.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+```
+npm install --save-dev riot-testing-library
+```
+
+```javascript
+import render, { cleanup, fireEvent } from 'riot-testing-library'
+import TestTag from './test.tag'
+
+afterEach(() => {
+  cleanup()
```
Comment (kentcdodds) [inline docs/ecosystem-riot-testing-library.md]: Have you considered [making this cleanup automatic](https://testing-library.com/docs/react-testing-library/setup#skipping-auto-cleanup) ([like React Testing Library](https://github.com/testing-library/react-testing-library/blob/89d11b08c61f3cf5af770c61a03d564b3fdc301c/src/index.js#L4-L14))?

File: docs/ecosystem-riot-testing-library.md (hunk 1 above)
Comment (ariesjia) [reply docs/ecosystem-riot-testing-library.md]: Thanks, this `async` is unnecessary,  I removed it

File: docs/ecosystem-riot-testing-library.md (hunk 2 above)
Comment (ariesjia) [reply docs/ecosystem-riot-testing-library.md]: This feature is awesome !
I don't pay attention to it when I use react-testing-library. It's easy to use.
I add this feature.

**Comment by kentcdodds** [comment]
@all-contributors please add @ariesjia for docs

**Pull request merged**